astroplan/constraints.py

```diff
@@ -546,6 +546,48 @@ def __init__(self, min=None, max=None, ephemeris=None):
         self.max = max
         self.ephemeris = ephemeris
 
+    @classmethod
+    def dark(cls, **kwargs):
+        """
+        This will use default values (acceptable fractional illumination)
+        of max=0.25 with no minimum. You can input min= or max= to change
+        the values and ephemeris= if a different ephemeris is desired,
+        by default it uses ``astropy.coordinates.solar_system_ephemeris.set``
```

Comment:
This is nitpicking: Could you add double backticks around parameters (e.g ``min`` and ``max``) and remove the lone = from them. Also I suggest to make those links to ``astropy.coordinates.solar_system_ephemeris`` by removing the extra backticks, (and you can also add the ~ in the front)